third_party/xla_client/computation_client.h

```diff
@@ -166,10 +167,25 @@ class ComputationClient {
   // will be populated in an asynchrounous fashion.
   virtual DataPtr CreateDataPlaceholder(std::string device, Shape shape) = 0;
 
+  // Create DataPtr that only has dummy information which can be filled in
+  // later.
+  virtual std::vector<DataPtr> CreateAsyncDatas(
```

Comment:
nit. `Datas` to `Data` ?

Reply:
hmm, it takes vector or tensors and return vecotr of DataPtrs. Datas seems fine? wdyt?

Reply:
I should update `Create DataPtr` to `Create DataPtrs` through.. I can do that on master instead

Reply:
Sounds good, `DataPtrs` would make sense to describe the vector, as data is already plural :p Let's do naming changes later in master. Thank you @JackCaoG